health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/CampaignBoundary.js

```
@@ -0,0 +1,798 @@
+
+                        <SubBoundaryView
+                            title={bHierarchy[1]}
+                            arr={parent_group[bHierarchy[1]][0]}
+                        />
+
+                    </Card>) : (null)
+            }
+
+            {console.log("bHierarchy", bHierarchy)};
+
+            {
```

Biome

[error] There is a suspicious semicolon in the JSX element.

This is usually the result of a typo or some refactor gone wrong.
Remove the semicolon, or move it inside a JSX element.

(lint/suspicious/noSuspiciousSemicolonInJsx)